extensions/plugins/system/redcore/redcore.php

```diff
@@ -316,6 +316,18 @@ public function onBeforeCompileHead()
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.js']);
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.min.js']);
 			}
+
+			// Remove permission.js. Specially for Joomla 3.6
+			if ($isAdmin && version_compare(JVERSION, '3.6.0', '>='))
```

Comment:
I think this should fire with additional checks like option == 'com_redcore' && view == 'config' or it will remove permissions script even for Joomla config